### PR TITLE
6 get api articles article comments

### PR DESCRIPTION
**Pull request by jackharbon**
- app.test -> added tests for GET /api/articles/:article_id/comments
- errors-controller - removed console.logs
- app -> added getCommentsByArticleId
- articles-controller -> added getCommentsByArticleId/selectCommentsByArticleId
- articles-models -> added selectCommentsByArticleId

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -113,3 +113,38 @@ describe('GET: /api/articles/:article_id', () => {
...
+			.then((response) => {
+				expect(response.body.comments).toEqual(expect.any(Array));
+				expect(Object.keys(response.body.comments[0])).toEqual(
+					expect.arrayContaining(['body', 'votes', 'author', 'article_id', 'created_at'])
+				);
+				response.body.comments.forEach((comment) => {
+					expect(comment.article_id).toBe(1);
+				});
+			});
+	});
```
Comment (Geo-Beard) [inline __tests__/app.test.js:131]: Add in a test for: Status 200, valid ID, but has no comments responds with an empty array of comments.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -113,3 +113,38 @@ describe('GET: /api/articles/:article_id', () => {
...
+		const articleId = 1;
+		return request(app)
+			.get(`/api/articles/${articleId}/comments`)
+			.expect(200)
+			.then((response) => {
+				expect(response.body.comments).toEqual(expect.any(Array));
+				expect(Object.keys(response.body.comments[0])).toEqual(
+					expect.arrayContaining(['body', 'votes', 'author', 'article_id', 'created_at'])
+				);
+				response.body.comments.forEach((comment) => {
```
Comment (Geo-Beard) [inline __tests__/app.test.js:127]: Need to check that the comments are coming back in the correct order - jest-sorted will be useful here.

**Review by Geo-Beard — COMMENTED** [review]
Looks good overall - just a couple of comments to respond to - implement those changes then feel free to merge and move on :)

File: __tests__/app.test.js (hunk 2 above)
Comment (jackharbon) [reply __tests__/app.test.js:127]: jest-sorted doesn't work here, it needs to add
`"jest": {
  "setupFilesAfterEnv": ["jest-sorted"]
}`
while there's already a line in `package.json`
`"jest": {
		"setupFilesAfterEnv": [
			"jest-extended/all"
		]
	}` 
I don't know how to connect jest-sorted
